CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ All notable changes to this project will be documented in this file. The format
 
 - Changed:
 
+  - Upgrade the public + partners apps to Next v10 along with new architectural patterns [#1087](https://github.com/bloom-housing/bloom/pull/1087))
```

Comment:
Can we start separating the changes with release headers, so if we can agree the new version should be v1.0.0, to indicate major breaking changes can we add at the top of this section a header like `## [1.0.0] - 2021-05-20`? So maybe replace `### General` with that? What do you think @jaredcwhite and @pbn4 ?